android/app/src/main/java/com/zulipmobile/SentryUtils.kt

```diff
@@ -0,0 +1,26 @@
+package com.zulipmobile
+
+import io.sentry.Sentry
+import io.sentry.SentryLevel
+
+/**
+ * A home for things that ought to be static extensions of `Sentry`.
+ *
+ * Extending Java classes with static members isn't currently a feature
+ * available in Kotlin:
+ *   https://youtrack.jetbrains.com/issue/KT-11968
+ * so this is our substitute.
+ */
+class SentryX {
+    companion object {
+        /**
+         * Like `Sentry.captureException`, but at level `SentryLevel.WARNING`.
+         */
+        public fun warnException(e: Throwable) {
+            Sentry.withScope { scope ->
+                scope.level = SentryLevel.WARNING
+                Sentry.captureException(e)
```

Comment:
Interesting, I think I'd expect that you'd have to pass the `scope` param somewhere in the `Sentry.captureException` call. But I guess you don't: https://docs.sentry.io/platforms/java/enriching-events/scopes/#local-scopes

Reply:
Yeah. I think this API actually makes more sense than it would if you did have to pass it: basically the point is that there's always a current scope (there has to be, to handle uncaught exceptions where you don't explicitly invoke Sentry at all), and so `Sentry.captureException` uses that.

The `withScope` function is sort of doing two things at once: it pushes a new scope onto the scope stack (and undertakes to pop it at the end), and then it passes you the scope much like `Sentry.configureScope` does.  I think that's a bit confusing, and if I were designing this API, I'd probably separate those, so that you'd write:
```kt
Sentry.pushScope {
  Sentry.configureScope { scope -> scope.level = SentryLevel.WARNING }
  Sentry.captureException(e)
}
```

(Well, and I'd probably make `configureScope` a bit less cumbersome too, like:
```kt
Sentry.pushScope {
  Sentry.currentScope.level = SentryLevel.WARNING
  Sentry.captureException(e)
}
```
)